### ISSUE TITLE
[Feature Request]: categorized templates

### ISSUE THREAD

**Issue by arashsheyda** [nuxt/nuxt.com#1445]
It would be fantastic to have the ability to categorize templates, similar to modules

**Comment by Atinux** [comment]
Agree, but when we will have too many templates that will need to be categorised :)
